client/gutenberg/extensions/editor-notes/index.js

```diff
@@ -3,6 +3,7 @@
 /**
  * External dependencies
  */
+import React from 'react';
```

Comment:
We'll probably have to do this for the rest of the blocks too..

How about we use `webpack.ProvidePlugin` to ask Webpack to load React without having to specifically `import` it?